src/directory/directory.js

```diff
@@ -363,6 +368,11 @@ const directory = {
             route: '/lib/auth/escapehatch',
             filters: ['android', 'ios']
           },
+          {
```

Comment:
Same thing here for advanced workflow?

Reply:
If I use the existing directory entry, it messes up the order.. For example: 

`Advance workflows` would show up before `Sign in` which we don't want.